timesketch/frontend-ng/src/components/Explore/EventList.vue

```diff
@@ -986,6 +1002,35 @@ export default {
           this.isSummaryLoading = false
         })
     },
+    generateForensicReport() {
```

Comment:
Do you really want to call it a `Forensic Report`? The term could be misunderstood, so maybe something more specific to what this function is doing: `keyEventReport`?